src/components/CurrencyInputPanel/index.tsx

```diff
@@ -92,6 +91,7 @@ const CurrencySelect = styled(ButtonGray)<{
     background-color: ${({ selected, theme }) => (selected ? theme.deprecated_bg3 : darken(0.05, theme.accentAction))};
   }
   visibility: ${({ visible }) => (visible ? 'visible' : 'hidden')};
+  ${({ labelOnly }) => labelOnly && `pointer-events: none`}
```

Comment:
nit: with this being so direct an effect I think you could change the arg name from `labelOnly` to be `pointerEvents` which takes in a string and it would be more clear what is the effect.

Reply:
Thanks that's a great suggestion. Pushed a new commit with the rename.